example/bug/service.pb.gorm.go

```diff
@@ -0,0 +1,346 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: example/bug/service.proto
+
+/*
+Package bug is a generated protocol buffer package.
+
+It is generated from these files:
+	example/bug/service.proto
+
+It has these top-level messages:
+	AclItem
+	Acl
+*/
+package bug
+
+import (
+	context "context"
+
+	errors "errors"
+
+	gorm "github.com/jinzhu/gorm"
+
+	"github.com/Infoblox-CTO/ngp.api.toolkit/mw/auth"
+	ops "github.com/Infoblox-CTO/ngp.api.toolkit/op/gorm"
+	proto "github.com/gogo/protobuf/proto"
+
+	fmt "fmt"
+
+	math "math"
+
+	_ "github.com/infobloxopen/protoc-gen-gorm/options"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// ACLItemORM no comment was provided for message type
+type ACLItemORM struct {
+	ID uint64
+}
+
+// TableName overrides the default tablename generated by GORM
+func (ACLItemORM) TableName() string {
+	return "acl_items"
+}
+
+// ConvertACLItemToORM takes a pb object and returns an orm object
+func ConvertACLItemToORM(from AclItem) (ACLItemORM, error) {
+	to := ACLItemORM{}
+	var err error
+	to.ID = from.Id
+	return to, err
+}
+
+// ConvertACLItemFromORM takes an orm object and returns a pb object
+func ConvertACLItemFromORM(from ACLItemORM) (AclItem, error) {
+	to := AclItem{}
+	var err error
+	to.Id = from.ID
+	return to, err
+}
+
+// ACLORM no comment was provided for message type
+type ACLORM struct {
+	TenantID string
+	ID       uint64
+	ACLItems []*AclItemORM `gorm:"foreignkey:ACLID"`
```

Comment:
This line should be: `ACLItems []*ACLItemORM `gorm:"foreignkey:ACLID"`

Either that, or the struct should be defined as `AclItemORM` the same way the regular `pb.go` structs are named.